src/main/scala/cromwell/binding/Call.scala

```diff
@@ -34,6 +35,27 @@ case class Call(alias: Option[String], task: Task, inputMappings: Map[String, Wd
    * @return Map[String, WdlType] representing each task-local input
    */
   def unsatisfiedInputs: Map[String, WdlType] = task.inputs.filterNot { case (k, v) => inputMappings.contains(k) }
-  
+
   override def toString: String = s"[Call name=$name, task=$task]"
+
+  /** Find all calls upon which this call immediately depends, i.e. the result of this
+    * does not include transitive dependencies.  Currently this only works for member
+    * access expressions with a literal LHS, e.g.:
+    * 
+    * {{{
+    *   call cgrep {
+    *     input: in_file=ps.procs
+    *   }
+    * }}}
+    * 
+    */
+  def prerequisiteCalls(): Traversable[Call] = {
```

Comment:
Actually thinking about this a bit more, I think the mature version of `prerequisiteCalls` probably should return `Set[Call]`.  It's possible that multiple inputs could express dependencies on the same parent:
```
call a
call b {
  input: input1 = a.output1, input2 = a.output2
}
```
But if all we're returning are `Call`s and not some mapping from input names to `Call`s, this returned collection should probably not have duplicates.  This isn't an issue for the 3-step workflow, but this might as well come closer to a correct API signature now. :smile: